### ISSUE TITLE
Add check for javascript enabled

### ISSUE THREAD

**Issue by aschey** [ClipboardProject/ClipboardApp#147]

**Comment by somersbmatthews** [comment]
Do you mean like '<noscript>' javascript disabled check ? There is html sanitization in next and vue that prevent '<noscript>' elements. This can be overridden with security risks or can be implemented in a wierd way in 'nuxt.config.js'. I will make WIP PR to demonstrate if this kind of thing is what you mean.

**Comment by aschey** [comment]
Yeah, I think we just want some kind of UI element that shows if they have javascript disabled and says "javascript is required to run this app" or something. I think we'll have to do it using `nuxt.config.js`.
